packages/dapp/src/helpers/config.ts

```diff
@@ -13,7 +13,6 @@ export interface ConfigType {
   SENTRY_DSN?: string;
   STRIPE_CLIENT_ID?: string;
   STRIPE_API_KEY?: string;
-  APP_VERSION?: string;
```

Comment:
This is not supplied by env variables in neither staging nor prod. I grepped repos for whole joincivil organization and didn't see `APP_VERSION` used anywhere.

Reply:
It's currently referenced only one place, in the Sentry init config, where it's currently undefined, which is identical to not including the key in the first place (unless Sentry is doing something very weird iterating over keys).

But I agree this could be useful if the container supplied a value for it, I'll leave it in for now.